data/sql/updates/pending_db_world/Update_quest_6002.sql

```diff
@@ -0,0 +1,23 @@
+(6002, 2, -1, 10, -263, 0, 0, 0, 0, 0, 0, 0, 5, 0, 0, 132300, 0, 19179, 0, 0, 15710, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 609, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 32, 'Body and Heart', 'Use the Cenarion Lunardust on the Moonkin Stone between Mulgore and the Barrens to bring forth Lunaclaw.  From there, you must face Lunaclaw and earn the strength of body and heart it possesses.$B$BSpeak with Turak Runetotem in Thunder Bluff when you are done.', 'On land west of Taurajo in the Barrens - just before it turns into Mulgore - lies a Moonkin Stone.  The moonkin are special beasts, not native to these parts.  Lunaclaw, the stone\'s defender, will face you when Cenarion Lunardust is applied to it.  Lunaclaw possesses within it a strength that you must use as one of the Claw.$B$BYou\'ve earned the right to use this Cenarion Lunardust, $N... now show me that you\'ve earned the right to possess the strength of body and the strength of heart needed as a druid!', 'Face Lunaclaw and earn the strength of body and heart it possesses.', 'Return to Turak Runetotem in Thunder Bluff.', 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '', '', '', '', 52237);
+-- Update table quest_request_items
+DELETE FROM `quest_request_items` WHERE (`ID` = 6002);
+INSERT INTO `quest_request_items` (`ID`, `EmoteOnComplete`, `EmoteOnIncomplete`, `CompletionText`, `VerifiedBuild`) VALUES
+(6002, 1, 0, 'When you have faced the challenge that lies before you, your understanding of strength of body and strength of heart will be fully realized.  Until that time, I cannot help you further.', 52237);
+-- Update POi
+DELETE FROM `quest_poi` WHERE `QuestID`=6002;
+INSERT INTO `quest_poi` (`QuestID`, `id`, `ObjectiveIndex`, `MapID`, `WorldMapAreaId`, `Floor`, `Priority`, `Flags`, `VerifiedBuild`) VALUES 
+(6002, 0, -1, 1, 1639, 0, 0, 3, 52237),
+(6002, 1, 16, 1, 1156, 0, 0, 1, 52237);
```

Comment:
If you feel comfortable, please send me the packet you found this in. It does not match 51831.

Reply:
no problem, here it is:

[dump_3.4.3.52237_2023-12-06_00-58-29_parsed.txt](https://github.com/azerothcore/azerothcore-wotlk/files/13593872/dump_3.4.3.52237_2023-12-06_00-58-29_parsed.txt)

Reply:
> no problem, here it is:

```
ServerToClient: SMSG_QUEST_POI_QUERY_RESPONSE (0x2A9D)
[...]
[0] QuestID: 6002
[...]
[0] [1] ObjectiveIndex: 30
```
This does not match the data in your PR.

Reply:
> > no problem, here it is:
> 
> ```
> ServerToClient: SMSG_QUEST_POI_QUERY_RESPONSE (0x2A9D)
> [...]
> [0] QuestID: 6002
> [...]
> [0] [1] ObjectiveIndex: 30
> ```
> 
> This does not match the data in your PR.

I kept this 16 exactly because the quest doesn't work with 30, I've tested it countless times (always deleting the wow cache folder to be sure), it seems to follow a certain pattern for several of them, but I kept the value 16 for the purposes of normal Quest functionality.

Reply:
> Same problem with your quest_template adjustment then, why are you putting a VerifiedBuild value for something that contradicts the build you propose it matches?

for the same reason that the current build also feeds on erroneous data probably taken from another database (TrinityCore or SunwellCore), so finding a compromise to make the mission work correctly on the client side was not a bad idea, even knowing that some values ​​differ from sniff values.